src/MobileApps/MyDriving/MyDriving.UWP/MyDriving.UWP.csproj

```diff
@@ -400,7 +400,7 @@
     <SDKReference Include="Microsoft.VCLibs, Version=14.0">
       <Name>Visual C++ 2015 Runtime for Universal Windows Platform Apps</Name>
     </SDKReference>
-    <SDKReference Include="SQLite.UWP.2015, Version=3.12.1">
+    <SDKReference Include="SQLite.UWP.2015, Version=3.12.2">
```

Comment:
We need to get this from Thiago and update it ourselves. I think its a perfect segway to that action.